index.js

```diff
@@ -262,6 +274,8 @@ function Argv (processArgs, cwd) {
         self.default(key, opt.default)
       } if ('nargs' in opt) {
         self.nargs(key, opt.nargs)
+      } if ('choices' in opt) {
```

Comment:
nice, we should definitely loop around and add a few other missing features to this shorthand.